doc/start/quick-cephfs.rst

```diff
@@ -3,117 +3,130 @@
+different nodes::
 
-	cat ceph.client.admin.keyring
+    ceph osd crush rule create-replicated {rule_name} default osd
+    ceph osd pool set data crush_rule (rule_name} data
+    ceph osd pool set data crush_rule {rule_name} meta
 
-#. Copy the key of the user who will be using the mounted CephFS file system.
-   It should look something like this:: 
-	
-	[client.admin]
-	   key = AQCj2YpRiAe6CxAA7/ETt7Hcl9IyxyYciVs47w==
+Usually, setting ``pg_num`` to 32 gives a perfectly healthy cluster. If you need
+to pick appropriate value for ``pg_num`` based on your needs, refer
+`Placement Group`_ reference. Nautilus release introduced pg_autoscaler plugin
+which can automatically increase/decrease value of ``pg_num``; to use it refer
+the `Placement Group`_ reference.
 
-#. Open a text editor. 
+Mounting the File System
+========================
 
-#. Paste the key into an empty file. It should look something like this::
+Using Kernel Driver
+-------------------
 
-	AQCj2YpRiAe6CxAA7/ETt7Hcl9IyxyYciVs47w==
+The command to mount CephFS using kernel driver usually looks like this::
 
-#. Save the file with the user ``name`` as an attribute 
-   (e.g., ``admin.secret``).
-
-#. Ensure the file permissions are appropriate for the user, but not
-   visible to other users. 
-
-
-Kernel Driver
-=============
+	sudo mount -t ceph {ip-address-of-MON}:{port-number-of-MON}:{path-to-be-mounted} -o name={user-name},secret={secret-key} {mount-point}
 
-Mount CephFS as a kernel driver. ::
+The IP address of the monitor is usually present in the ``ceph.conf``,
+the port number needs to found out, ``{path-to-be-mounted}`` is often root (of
+the CephFS), ``{mount-point}`` is the directory in your file system where
+CephFS would be mounted. The argument passed right after the option ``-o`` are
+CephX credentials required to mount CephFS which, by default, is enabled.
+``{secret-key}`` for a user - admin in the following example - can obtained as
+follows::
 
-	sudo mkdir /mnt/mycephfs
-	sudo mount -t ceph {ip-address-of-monitor}:6789:/ /mnt/mycephfs
+    cat /etc/ceph/ceph.client.admin.keyring
+    [client.admin]
+        key = AQD9t39d3vdjMBAAOf4xhiJk+Te0cblzKeF+5A==
+        caps mds = "allow *"
+        caps mgr = "allow *"
+        caps mon = "allow *"
+        caps osd = "allow *"
 
-The Ceph Storage Cluster uses authentication by default. Specify a user ``name``
-and the ``secretfile`` you created  in the `Create a Secret File`_ section. For
-example::
+    sudo mount -t ceph 192.168.0.1:6789:/ /mnt/mycephfs -o name=admin,secret=AQD9t39d3vdjMBAAOf4xhiJk+Te0cblzKeF+5A==
 
-	sudo mount -t ceph 192.168.0.1:6789:/ /mnt/mycephfs -o name=admin,secretfile=admin.secret
+Alternatively, you can save the secret key in a file and pass the path to the
+file instead::
 
+    echo AQD9t39d3vdjMBAAOf4xhiJk+Te0cblzKeF+5A== > admin.secret
+    sudo mount -t ceph 192.168.0.1:6789:/ /mnt/mycephfs -o name=admin,secretfile=admin.secret
```

Comment:
I suspect much of the above verbiage can be removed. We should update this just let the mount helper find the mon addrs and secrets via the config file and keyring. With that you can just let them do this:

`sudo mount -t ceph :/ /mnt/mycephfs -o name=admin`

...and skip all of the stuff about putting the secret in a separate file. I'll leave it up to you whether you want to do that here or in a separate PR.

Reply:
Modified the doc. Please take a look.